lib/imperative-test.js

```diff
@@ -489,11 +491,11 @@ class ImperativeTest extends Test {
     });
   }
 
-  resolves(input, expected) {
+  resolves(input, expected = nothingSymbol) {
```

Comment:
Also, with this change there will be no way to check that the Promise resolves with the value of `undefined`.